From 1010b8394a23e282bcf126babf61e9c32657e54d Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Wed, 15 Aug 2018 17:53:39 +0800
Subject: [PATCH 4477/5242] MLK-19227-3: arm64:dts:imx8qm-lpddr4-arm2-hdmi:
 Remove fsl,cec

commit  7b5b41ad07a4668df4bbdb45e13b3478917250d4 from
https://source.codeaurora.org/external/imx/linux-imx.git

CEC and EDID are not supported by lpddr4 arm4 board.
So remove fsl,cec and add fsl,no_edid properity.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit fbf7da4423c74e59830cf460e8f06cbd174d7be8)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts
index f01ac97..8301c8f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-hdmi.dts
@@ -60,7 +60,7 @@
 	assigned-clock-parents = <&clk IMX8QM_HDMI_AV_PLL_CLK>,
 							<&clk IMX8QM_HDMI_AV_PLL_CLK>,
 							<&clk IMX8QM_HDMI_AV_PLL_CLK>;
-	fsl,cec;
+	fsl,no_edid;
 	status = "okay";
 };
 
-- 
1.7.9.5

